src/qiboml/models/ansatze.py

```diff
@@ -13,7 +13,7 @@ def HardwareEfficient(
     circuit = Circuit(nqubits)
 
     for _ in range(nlayers):
-        for q in qubits:
+        for q in range(qubits):
```

Comment:
```suggestion
        for q in qubits:
```
this is actually just qubits, because qubits is a tuple already